### DISCUSSION TITLE
issues in modelling fire spread in cables

### DISCUSSION THREAD

**Discussion by nabilmahmoudmohamed** [firemodels/fds#12861] [Q&A] [answered]
I'm trying to model fire spread in vertical & horizontal cables (with ignitor particle cloud at mid of cables) but i noted one or more of the following occurs:
fire keeps fluctuating/jumping between cables back and forth or flames may auto extinguish itself, please take a look of the fds record below and guide me if i am missing parameters or for fine tuning.

also does fluctuating flames means that i need to improve mesh quality and use smaller cells size?

lastly, i would like your kind support in guiding me towards available resources so that i can properly define used materials & related thermal properties and reactions of the actual used cable, cable is consisting of XLPE insulation, PVC sheath & Aluminums conductors.
attached files fds files for ref., thanks in advance.

--------------------

&MESH ID='MESH-01-merged', IJK=19,8,24, XB=0.0,0.95,3.2,3.6,0.35,1.55/
&MESH ID='MESH-03-merged', IJK=19,7,24, XB=0.0,0.95,3.6,3.975,0.35,1.55/
&MESH ID='MESH-05-merged', IJK=19,8,24, XB=0.0,0.95,3.975,4.375,0.35,1.55/
&MESH ID='MESH-07-merged', IJK=19,8,24, XB=0.0,0.95,4.375,4.75,0.35,1.55/
&MESH ID='MESH-09-merged', IJK=19,8,24, XB=0.0,0.95,4.75,5.15,0.35,1.55/
&MESH ID='MESH-11-merged', IJK=19,8,24, XB=0.0,0.95,5.15,5.525,0.35,1.55/
&MESH ID='MESH-13-merged', IJK=19,8,24, XB=0.0,0.95,5.525,5.925,0.35,1.55/

&SPEC ID='POLYURETHANE', FORMULA='C6.3H7.1N1.0O2.1'/

&PART ID='ignitor particle',
      STATIC=.TRUE.,
      SURF_ID='ignitor'/

&REAC ID='SFPE POLYURETHANE_GM27',
      FYI='SFPE Handbook, 5th Edition, Tables A.38 and A.39',
      FUEL='POLYURETHANE',
      CO_YIELD=0.042,
      SOOT_YIELD=0.198,
      HEAT_OF_COMBUSTION=1.64E+4,
      RADIATIVE_FRACTION=0.537/

&MATL ID='CABLE',
      FYI='Properties completely fabricated',
      SPECIFIC_HEAT=1.0,
      CONDUCTIVITY=0.1,
      DENSITY=1183.0,
      HEAT_OF_COMBUSTION=2.27E+4,
      N_REACTIONS=1,
      HEAT_OF_REACTION=800.0,
      SPEC_ID(1,1)='POLYURETHANE',
      NU_SPEC(1,1)=1.0,
      REFERENCE_TEMPERATURE=280.0/

&SURF ID='cable',
      FYI='Properties completely fabricated',
      RGB=128,200,173,
      BURN_AWAY=.TRUE.,
      BACKING='VOID',
      MATL_ID(1,1)='CABLE',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.1/
&SURF ID='ignitor',
      NET_HEAT_FLUX=50.0,
      EMISSIVITY=1.0,
      GEOMETRY='CYLINDRICAL',
      LENGTH=0.15,
      RADIUS=0.01/

&INIT ID='ignitor particle', PART_ID='ignitor particle', N_PARTICLES_PER_CELL=1, CELL_CENTERED=.TRUE., XB=0.1,0.8,4.711891,4.761891,0.488109,0.55/ 

&OBST ID='Obstruction #2', XB=0.1,0.2,3.311891,5.911891,0.55,0.65, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.3,0.4,3.311891,5.911891,0.55,0.65, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.5,0.6,3.311891,5.911891,0.55,0.65, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.7,0.8,3.311891,5.911891,0.55,0.65, BULK_DENSITY=1183.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.1,0.2,3.311891,5.911891,0.85,0.95, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.3,0.4,3.311891,5.911891,0.85,0.95, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.5,0.6,3.311891,5.911891,0.85,0.95, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.7,0.8,3.311891,5.911891,0.85,0.95, BULK_DENSITY=1183.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.1,0.2,3.311891,5.911891,1.15,1.25, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.3,0.4,3.311891,5.911891,1.15,1.25, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.5,0.6,3.311891,5.911891,1.15,1.25, BULK_DENSITY=40.0, SURF_ID='cable'/ 
&OBST ID='Obstruction #2', XB=0.7,0.8,3.311891,5.911891,1.15,1.25, BULK_DENSITY=1183.0, SURF_ID='cable'/ 

&VENT ID='Mesh Vent: Mesh-01 [YMAX]', SURF_ID='OPEN', XB=0.0,0.95,5.925,5.925,0.35,1.55/ 
&VENT ID='Mesh Vent: Mesh-06 [YMIN]', SURF_ID='OPEN', XB=0.0,0.95,3.2,3.2,0.35,1.55/ 
&VENT ID='Mesh Vent: Mesh-06 [ZMAX]', SURF_ID='OPEN', XB=0.0,0.95,3.2,5.925,1.55,1.55/

**Comment by TristanHehnen** [comment, marked as answer]
To the material definitions:
This is a somewhat complex topic. Depending on what you would like to achieve, you may get fairly good results with simplified models.

The following approaches could be considered:
1. the FLASH-CAT model from the CHRISTIFIRE campaign ([phase 1](https://www.nist.gov/publications/cable-heat-release-ignition-and-spread-tray-installations-during-fire-christifire-0), [phase 2](https://www.nist.gov/publications/cable-heat-release-ignition-and-spread-tray-installations-during-fire-christifire-phase))
2. the [scaled pyrolysis ](https://doi.org/10.1016/j.firesaf.2023.103980) approach by @johodges et al. -- implemented in FDS as S-Pyro
3. create material parameter sets with thermal decomposition from experiments

Creating the material parameter sets  is fairly complex, a lot of effort went already into researching this for cables alone by many different research groups. To properly define your material and pyrolysis models, I think it would be best if you could get access to cable samples and run the required experiments (DSC, TGA, MCC, cone) yourself. Otherwise you may have trouble getting consistent data sets. The effort in creating a parameter set for a specific material can be quite large. 

I think S-Pyro is promising and also more flexible than FLASH-CAT. 
My suggestion would be to look into S-Pyro first.

**Reply by h7899** [reply]
I AM A BEGINNER, PLS HOW CAN I GET A FLASH-CAT MODEL CASE.

**Reply by drjfloyd** [reply]
FLASH-CAT is a method for developing a time dependent heat release rate. It isn't something built into FDS. Read NUREG/CR-7010 for details on FLASH-CAT.
